blog/2024-06-03-ratify-1.2.0.mdx

```diff
@@ -0,0 +1,71 @@
+---
+slug: ratify-v1.2.0-is-available
+title: Ratify v1.2.0 is available! 
+authors: [ratify]
+tags: [verification]
+---
+
+We are thrilled to announce the release of Ratify v1.2.0! 
+
+Ratify is a comprehensive verification framework designed to ensure that all supply chain artifacts stored in a registry are thoroughly vetted and trustworthy. By integrating Ratify into your workflow, you can safeguard your deployment pipeline from potential threats and vulnerabilities, enhancing the overall security of your operations. To see an end to end example of how Ratify can be used, check out this Microsoft Build 2024 session: [Securing the Containers’ Supply Chain for Azure Kubernetes Service](https://youtu.be/5W-22xU4EP0?si=EMlmTDXRzFHj0zw0). 
+
```

Comment:
@FeynmanZhou , i m wondering if we should put a more vendor neutral demo.  Here is a one recorded by Akash, https://www.youtube.com/watch?v=8caLL0mz9ks.

Reply:
Yes. In the announcement blog post, it would be better to keep the content vendor-neutral and avoid routing to a  single vendor.

Reply:
Great suggestion @susanshi, I'll update the link.

Reply:
I would suggest adding a section "Ratify ecosystem" to elaborate and list partners like AWS, Venafi, Microsoft Azure, etc. Here is a sample: https://notaryproject.dev/blog/2023/announcing-major-release/#extensibility-plugin-support-for-notation

Reply:
Thanks for the link @FeynmanZhou, I'll include that as well to give new users some resources to get an overview.

Reply:
I updated the text to link to the article linked by @FeynmanZhou. Adding another link to the video made it too busy and distracted from the main call to action which is to read the announcement post. So, I decided not to include a link to any videos. Happy to change that if people disagree with that decision.